owncloudApp/src/main/java/com/owncloud/android/db/PreferenceManager.java

```diff
@@ -48,16 +50,18 @@
     private static final String AUTO_PREF__SORT_ORDER_UPLOAD = "sortOrderUpload";
     private static final String AUTO_PREF__SORT_ASCENDING_UPLOAD = "sortAscendingUpload";
 
-    public static final String PREF__CAMERA_PICTURE_UPLOADS_ENABLED = "camera_picture_uploads";
-    public static final String PREF__CAMERA_VIDEO_UPLOADS_ENABLED = "camera_video_uploads";
-    public static final String PREF__CAMERA_PICTURE_UPLOADS_WIFI_ONLY = "camera_picture_uploads_on_wifi";
-    public static final String PREF__CAMERA_VIDEO_UPLOADS_WIFI_ONLY = "camera_video_uploads_on_wifi";
+    public static final String PREF__CAMERA_PICTURE_UPLOADS_ENABLED = "enable_picture_uploads";
+    public static final String PREF__CAMERA_VIDEO_UPLOADS_ENABLED = "enable_video_uploads";
+    public static final String PREF__CAMERA_PICTURE_UPLOADS_WIFI_ONLY = "picture_uploads_on_wifi";
+    public static final String PREF__CAMERA_VIDEO_UPLOADS_WIFI_ONLY = "video_uploads_on_wifi";
     private static final String PREF__CAMERA_UPLOADS_ACCOUNT_NAME = "camera_uploads_account_name";  // NEW - not
     // saved yet
-    public static final String PREF__CAMERA_PICTURE_UPLOADS_PATH = "camera_picture_uploads_path";
-    public static final String PREF__CAMERA_VIDEO_UPLOADS_PATH = "camera_video_uploads_path";
-    public static final String PREF__CAMERA_UPLOADS_BEHAVIOUR = "camera_uploads_behaviour";
-    public static final String PREF__CAMERA_UPLOADS_SOURCE = "camera_uploads_source_path";
+    public static final String PREF__CAMERA_PICTURE_UPLOADS_PATH = "picture_uploads_path";
+    public static final String PREF__CAMERA_VIDEO_UPLOADS_PATH = "video_uploads_path";
+    public static final String PREF__CAMERA_PICTURE_UPLOADS_BEHAVIOUR = "picture_uploads_behaviour";
+    public static final String PREF__CAMERA_PICTURE_UPLOADS_SOURCE = "picture_uploads_source_path";
+    public static final String PREF__CAMERA_VIDEO_UPLOADS_BEHAVIOUR = "video_uploads_behaviour";
+    public static final String PREF__CAMERA_VIDEO_UPLOADS_SOURCE = "video_uploads_source_path";
```

Comment:
I understand that new names are more descriptive. The point here is that it will break the old camera upload configuration.

This is an important point. What should we expect when migrating the old camera uploads configuration to the new one.

If we configured them in 2.17, they will be reset in 2.18. Users will need to enable them again and configure them again. Is this the expected behavior? Let's discuss it here: https://github.com/owncloud/android/issues/3130#issuecomment-819655805